manual/docs/build-and-deploy.md

```diff
@@ -0,0 +1,214 @@
+
+Projects that are independent of cfgov-refresh will need to provide their own deployment process, and all jobs they require. They will not be automatically included in the cf.gov deployment process, and will not be on the cf.gov release cadence. 
+
+All new deployment jobs, jobs that run in addition to the cf.gov depoyment jobs, as well as independent jobs, must be implemented with Jenkins-as-code.
+
+### Deployment QA
+
+All code that gets merged into the cfgov-refresh master must have adequate tests, as appropriate for the nature of the code. 
+
+This could potentially include unit tests, browser tests, and 508-compliance tests.
+
+There should be no drop in test coverage for cfgov-refresh.
+
+Regular releases of cf.gov on our release cadence are automated, presuming:
+
+- All unit tests pass 
+- All functional tests pass
+- There is no reduction in test coverage from the last release tagging
+
+### Current deployment process
+
+The current cf.gov deployment process requires running the following individual Jenkins jobs manually, in the following order:
+
+- cf.gov-beta-frontend-build, build with the tag name of the latest release, i.e. 3.5.2.
+- cf.gov-fab-deploy-django, build with DEPLOY_ENV Staging
+- cf.gov-fab-deploy-django, build with DEPLOY_ENV ***PRODUCTION***
+
+The following teams and individuals have the necessary access to deploy cf.gov releases:
+
+- Ross Karchner
+- Scott Cranfill
+- Will Barton
+- Bill Higgins
+- Serghei Gorobet
+- The Software Delivery team
+
+### Feature flags
+
+Feature flags are stored in the Wagtail database and exposed in the Wagtail admin under “Settings”:
+
+![Feature flags](img/image02.png)
+
+Here flags can added, enabled, or disabled:
+
+![Feature flags](img/image00.png)
+
+
+The flag consists of a single string (by convention all uppercase, with underscores instead of whitespace). Flags are disabled by default, and checks for flags that do not exist will return False.
+
+Once a flag has been created it can be checked in Python and Jinja2 templates using the functions flag_enabled and flag_disabled found in flags.template_functions and exposed to templates in v1. 
+
+#### Python example
+
+```python
+from flags.template_functions import flag_enabled, flag_disabled
+
+If flag_enabled('BETA_NOTICE'):
+	print(“Beta notice banner will be displayed”)
+
+If flag_disabled(‘BETA_NOTICE’):
+	print(“Beta notice banner will not be displayed”)
+```
+	
+Jinja2 example:
+
+```
+{% if flag_enabled('BETA_NOTICE') and show_banner %}
+    <div class="m-global-banner">
+I’m a beta banner.   
+    </div>
+{% endif %}
+```
+
+Feature flags are implemented as a Django app in cfgov-refresh/cfgov/flags. 
+
+#### Feature Flag Hygiene
+
+Feature flags should be rare and ephemeral. Changes should be small and frequent, and not big-bang releases, and flags that are no longer used should be cleaned up and removed from code and the database.
+
+### Wagtail pages vs. Django views
```

Comment:
This is one of several sections I envisioned being broken out into separate files when we were done with the working google doc for this. I can take care of that.

We might also want to fill it out with more useful content as we convert the jobmanager Django app to Wagtail.